libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -1332,7 +1332,7 @@ uint32_t NavEKF2::getLastVelNorthEastReset(Vector2f &vel) const
 const char *NavEKF2::prearm_failure_reason(void) const
 {
     if (!core) {
-        return nullptr;
+        return "EKF2 not enabled";
```

Comment:
This should be "not started", not having a core doesn't mean it wasn't enabled.

Reply:
why else would there be no core?